site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -132,10 +132,9 @@ assumed to have failed if its `conditions.Status.Ready` is not `True` for at lea
 seconds or if the node is missing (removed from the cluster). 
 
 Note that finding a replacement node within the old domain (like rack) may not always 
-be possible. Hence, we recommend using [WaitForPodsReady](link) and configuring `waitForPodsReady.recoveryTimeout`, to prevent 
-the workloads from waiting for the replacement indefinetly.
-
-
+be possible. Hence, we recommend using [WaitForPodsReady](site/content/en/docs/tasks/manage/setup_wait_for_pods_ready.md)
```

Comment:
You may double check if this link renders well, checking some other places we prefix the link with `/docs/tasks/manage/`.